packages/tools-iottwinmaker/src/commands/test-constants.ts

```diff
@@ -74,6 +75,48 @@ export const componentType1Definition = {
   description: getComponentType1Resp.description,
 };
 
+export const componentTypeWithInheritedProperty = {
```

Comment:
yes we'll want to keep all things typed :+1:

Reply:
I fixed most of the missing types in this file, but unfortunately there are some that don't belong to a specific "GetComponentTypeCommandOutput" or something simple like that because they have modified fields to fit tmdt's needs (differs from API types).

I could explicitly declare the full type definition e.g:

type componentTypeWithInheritedProperty: {
    componentTypeId: string | undefined;
    description: string | undefined;
    propertyDefinitions: {
        testProperty: {
            isInherited: boolean;
        };
    };
}

However that same information is visible when you hover over the name in TS, and this type would only be used 1 time, is this still necessary in these specific cases?

Reply:
Yes makes it even more important. precisely what are the differences? There's utility functions in typescript that can be used to slightly modify an existing type, so you will not need to fully re-declare the type.

see https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
Got it, will update!